src/index.html

```diff
@@ -1,11 +1,25 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
-</head>
-<body>
-    <h1>Hello webpack!</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Meals App</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header>
+      <div class="logo-div">
+        <a href="index.html"><img class="img-logo" src="./assets/logo-transparent.png" alt="meals" /></a>
```

Comment:
![1](https://user-images.githubusercontent.com/43011742/189833418-563b123e-373e-4cf8-b928-fecfe3116dde.PNG)
I am serving the app using the webpack server, but it does not display the images.
Here if you open the index.html file from the src folder with the live server, it does display the images but not with the webpack server. Have a look at this and make sure the webpack uses your HTML as the template.